src/nameresolver.nim

```diff
@@ -42,34 +42,41 @@ proc fillPackageLookupTable(c: var AtlasContext) =
     for entry in plist:
       c.urlMapping[unicode.toLower entry.name] = entry.url
 
-proc resolveUrl*(c: var AtlasContext; p: string): PackageUrl =
-  proc lookup(c: var AtlasContext; p: string): string =
+proc resolveUrl*(c: var AtlasContext; p: string): (PackageName, PackageUrl) =
+  proc lookup(c: var AtlasContext; p: string): tuple[name: string, url: string] =
     fillPackageLookupTable(c)
 
     if p.isUrl:
+      echo "IS URL: ", p
```

Comment:
debug leftover

Reply:
Ah thanks, yah I thought this PR was done but been playing a bit of whack-a-mole. I'll go through and clean it up.